### ISSUE TITLE
How to attach JavaDoc .jar for library .jar and get editor to show the documentation?

### ISSUE THREAD

**Issue by mattflix** [Microsoft/vscode-java-pack#105]
I have tried various techniques to specify the JavaDoc `.jar` that should be attached to a given library `.jar` (that appears in the "Referenced Libraries" section of the "JAVA DEPENDENCIES" view, and is successfully used by my app), but nothing has worked.

Currently, the only JavaDoc documentation that appears in the editor for me is that for the built-in objects/members, like `Object` and `toString()`.  No third-party libraries seem to have any JavaDoc exposed in the editor.

Is there any example of a simple project configured with a dependency on a library `.jar` (that has an associated JavaDoc `.jar`, also located on the local file system) where the editor is able to display the documentation comments for selected Java types and members?

**Comment by mattflix** [comment]
Btw, I have tried:
- Adding an matching `.properties` file next to the library `.jar` and setting a `"doc"` property to point at the JavaDoc `.jar`.
- Adding an `<attributes>` section for the library `.jar`'s entry in the `.classpath` file, with a `"javadoc_location"` attribute pointing at the JavaDoc `.jar`.

After each attempt to configure my project, I have tried both running the "Java: Refresh" command, closing and opening the folder (in VS Code), and restarting VS Code altogether.
